.gitattributes

```diff
@@ -0,0 +1,2 @@
+*.cabal linguist-generated
+/examples/init/foo.cabal -linguist-generated
```

Comment:
This is a hand-written file.
```suggestion
```

Reply:
To be clear, what I am saying here is that all `.cabal` files are generated, except for `/examples/init/foo.cabal` (note the minus sign in `-linguist-generated`).

Does this make sense?

Reply:
Ah, thanks for explaining! Looks good to me then.